### ISSUE TITLE
Output seems to be MAC address instead of IP

### ISSUE THREAD

**Issue by Elevennails** [vdjagilev/nmap-formatter#105]
Was outputting IP address using Docker container. However, I've installed using Go on a different device and now the outputs only seem to list MAC address.

**Comment by Elevennails** [comment]
Docker version was 1.0.0 seems to be an issue with 1.0.1

**Comment by vdjagilev** [comment]
Hi,

Thanks for using an app :) I've figured out why is this happening. I've managed to reproduce it by using `-O` option in nmap (fingerprint OS). An example of such XML output:

```xml
<address addr="192.168.1.1" addrtype="ipv4"/>
<address addr="FF:FF:FF:FF:FF:FF" addrtype="mac" vendor="[Snippet]"/>
```

In this case nmap-formatter takes only the last address. Will fix it.

**Comment by vdjagilev** [comment]
Fixed in [v1.0.2](https://github.com/vdjagilev/nmap-formatter/releases/tag/v1.0.2) bugfix release
@Elevennails
